macros/src/lib.rs

```diff
@@ -177,13 +186,20 @@ pub fn track(attr: TokenStream, item: TokenStream) -> TokenStream {
             /// Check for changes made to this struct with a given bitmask.
             ///
             /// To receive the bitmask, simply call `Type::#field_name()`
-            /// or `Type::#track_all()`;
+            /// or `Type::#track_all()`.
             #[warn(dead_code)]
             #[must_use]
             #struct_vis fn changed(&self, mask: #tracker_ty) -> bool {
                 self.tracker & mask != 0
             }
 
+            /// Check for any changes made to this struct.
+            #[allow(dead_code)]
+            #[must_use]
+            #struct_vis fn changed_any(&self) -> bool {
```

Comment:
I'm not sure whether this is a good name for the method, also because "any" is a valid field name. Maybe we just use `.changed()` or `.any_change()`

Reply:
`reset` could also be a name of a field, `track_all` too. No strong opinion though.

Reply:
> reset could also be a name of a field, track_all too. No strong opinion though.

This is different. If you call a field reset, you get methods called `get_reset` or `changed_reset`. If you call the field any you get the method `changed_any` twice, which would throw an error.

Reply:
I meant you could have field `reset` then you'll get a static method that will result in following errors:
```
22 | #[tracker::track]
   | ----------------- other definition for `reset`
23 | #[derive(Debug, Clone)]
24 | pub(in super::super) struct ProgressCircle {
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ duplicate definitions for `reset`

error[E0599]: the method `reset` exists for mutable reference `&mut ProgressCircle`, but its trait bounds were not satisfied
   --> src/frontend/widgets/progress_circle.rs:115:14
    |
24  | pub(in super::super) struct ProgressCircle {
    | ------------------------------------------ doesn't satisfy 10 bounds
...
115 |         self.reset();
    |              ^^^^^
    |              |
    |              this is an associated function, not a method
    |              field, not a method
```

Anyway, I'm happy to change to `any_change`, even though it will be less consistent with the rest of the names.

Reply:
You're right, I forgot about the methods for the bitfields.